openfisca_france/model/prelevements_obligatoires/taxe_habitation.py

```diff
@@ -65,13 +65,14 @@ class taxe_habitation(Variable):
 
     def function(self, simulation, period):
         period = period.start.offset('first-of', 'month').period('year')
+        last_year= period.start.offset('first-of', 'month').period(-1,'year')
         exonere_taxe_habitation = simulation.calculate('exonere_taxe_habitation', period)
         nombre_enfants_a_charge_menage = self.sum_by_entity(simulation.calculate('enfant_a_charge', period))
         nombre_enfants_majeurs_celibataires_sans_enfant = simulation.calculate('nombre_enfants_majeurs_celibataires_sans_enfant', period)
-        rfr_n_1_holder = simulation.compute('rfr_n_1', period)
+        rfr_holder = simulation.compute('rfr', last_year)
```

Comment:
but where ?
I'm really sorry but I'm not sure I get how to give period...
It is not working like that ?